core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -72,7 +72,7 @@
     {% if username %}
       <li ng-if="isLargeScreen" class="dropdown" popover-is-open="postTutorialHelpPopoverIsShown"
           popover-placement="bottom" popover-trigger="none"
-          popover="To get help in the future, click here!">
+          help-popover="To get help in the future, click here!">
```

Comment:
Nope, what you want to do in this case is to add an additional class, by doing `help-popover`, you may introduce new bugs later. An example of what I am talking about can be seen [here]( https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/about/about.html#L25). In addition, when you do so, please follow the [Oppia CSS conventions] (https://github.com/oppia/oppia/wiki/Coding-style-guide#css) and prefix your class with `oppia-`